app/src/main/java/com/ggetters/app/ui/central/sheets/AccountEditBottomSheet.kt

```
@@ -159,40 +161,120 @@ class AccountEditBottomSheet : BottomSheetDialogFragment() {
             }
             .setNegativeButton("Cancel", null)
             .show()
     }
     
+    private fun deleteUserAccount() {
+        lifecycleScope.launch {
+            try {
+                profileViewModel.deleteUserAccount()
+                Snackbar.make(requireView(), "Account deleted successfully", Snackbar.LENGTH_SHORT).show()
+                dismiss()
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)